.github/workflows/docs-manifest.yml

```diff
@@ -0,0 +1,40 @@
+name: Keep docs manifest up to date
+on:
```

Comment:
The `on` config here is what diverges the most from the other workflows. Initially, based on Gutenberg's components-changelog workflow, I wanted this check to be more conservative.

But now that I look at it again, and given the approach I finally took, maybe it should be the same as in the others. Thoughts?